algos/level2/week3/emin.index.js

```diff
@@ -0,0 +1,19 @@
+const get_score = (arr = [4, 2, 2, 3, 3, 4, 2]) => {
+	score = 0
+	level = 0
+	lineCounter = 0
+	basePoints = [40, 100, 300, 1200]
```

Comment:
A very good and clean solution in general but it fails when the player doesn't clear any line and we have "0" in our input array. In the instructions, the input range is given from 0 to 4. You seem to have ignored the "0" part also in your tests. ; ) Can you please edit your function and push it again?